tests/localization/field_tests.py

```diff
@@ -17,6 +18,11 @@
 # python manage.py test tests.localization.field_tests --settings="tests.test_settings"
 
 
+class I18nTextFieldFilterTests(ArchesTestCase):
+    def test_filter_related_model_i18n_text_field(self):
+        self.assertEqual(Node.objects.filter(graph__name__en="Graph Name").count(), 0)
```

Comment:
The other test classes in this file use custom models, but they didn't have foreign keys set up, so it seemed simpler  to just dogfood the real models. Open to changing this.